CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+# dbt_sage_intacct_source v0.1.1
+ 
+## Updates
+- Includes additional fields in the `gl_detail` model and excludes fivetran-deleted rows in the staging models. Adds variable configs for the AP Bill and AR Invoice related tables to allow users to disable them if not used.
```

Comment:
Can we be a bit more explicit as to what updates are being added. We should highlight which fields are being added and what variables have been included.